app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
+    <string name="request_leave">Yêu cầu đi trễ, về sớm</string>
+
+    <string name="employee_name">Tên nhân viên</string>
+    <string name="employee_code">Mã nhân viên</string>
+    <string name="project_name">Tên dự án</string>
+    <string name="position_name">Vị trí công việc</string>
+    <string name="branch">Chi nhánh</string>
+    <string name="group">Phòng ban</string>
+    <string name="contract_date"> Ngày bắt đầu làm việc chính thức</string>
+    <string name="start_probation_date">Ngày bắt đầu thử việc</string>
+    <string name="end_probation_date">Ngày kết thúc thử việc</string>
+    <string name="staff_type">Loại nhân viên</string>
+    <string name="position">Vị trí</string>
+    <string name="gender">Giới tính</string>
+    <string name="birthday">Ngày sinh</string>
+    <string name="off_have_salary_company_pay">Nghỉ có lương - Công ty trả</string>
+    <string name="number_day_off">Số ngày nghỉ</string>
+    <string name="off_no_salary">Nghỉ không lương</string>
+
+    <string name="off_type">Hình thức nghỉ</string>
+    <string name="annual_leave">Nghỉ phép thường niên (14.5 ngày)</string>
+    <string name="leave_for_marriage">Kết hôn (3.0 ngày)</string>
+    <string name="leave_for_child_marriage">Con kết hôn (1.0 ngày)</string>
+    <string name="funeral_leave">Nghỉ lễ tang (3.0 ngày)</string>
+    <string name="leave_for_care_of_sick_child">Con ốm (20.0 ngày)</string>
+    <string name="pregnancy_examination_leave">Nghỉ khám thai (5.0 ngày)</string>
+    <string name="sick_leave">Nghỉ đau ốm (60.0 ngày)</string>
+    <string name="miscarriage_leave">Nghỉ sẩy thai (50.0 ngày)</string>
+    <string name="maternity_leave">Nghỉ thai sản (180.0 ngày)</string>
+    <string name="wife_labor_leave">Vợ sinh con (14.0 ngày)</string>
+    <string name="company_pay">Công ty trả</string>
+    <string name="insurance_coverage">Bảo hiểm trả</string>
+    <string name="select_date">Chọn ngày</string>
+    <string name="select_time">Chọn thời gian</string>
+    <string name="reason">Lý do</string>
+    <string name="create">Tạo mới</string>
+    <string name="created_at">Thời gian tạo</string>
+    <string name="confirm_create">Xác nhận</string>
+    <string name="from">Từ</string>
+    <string name="to">Đến</string>
+    <string name="pending">Đang chờ</string>
+    <string name="accepted">Chấp nhận</string>
+    <string name="rejected">Từ chối</string>
+    <string name="forwarded">Chuyển tiếp</string>
+    <string name="leave_type">Hình thức xin phép</string>
+    <string name="compensation_time">Đăng ký thời gian làm bù</string>
+    <string name="example_time_from">Ví dụ: 2017/05/20 17:30</string>
+    <string name="example_time_to">Ví dụ: 2017/05/20 18:30</string>
+    <string name="confirm_request_leave">Xác nhận tạo yêu cầu đi trễ, về sớm</string>
+    <string name="submit">Gửi đi</string>
+    <string name="next">Tiếp theo</string>
+    <string name="status">Trạng thái</string>
```

Comment:
FATAL: [ExtraTranslation] The resource string "`status`" has been marked as `translatable="false"`
 If a string appears in a specific language translation file, but there is no corresponding string in the default locale, then this string is probably unused. (It's technically possible that your application is only intended to run in a specific locale, but it's still a good idea to provide a fallback.).

Note that these strings can lead to crashes if the string is looked up on any locale not providing a translation, so it's important to clean them up.